core/os/device/abi_test.go

```diff
@@ -35,7 +35,7 @@ var abiTestData = []struct {
 	{device.AndroidARMv7a, "armeabi-v7a", 32, 4, 4, 4, device.LittleEndian},
 	{device.AndroidARM64v8a, "arm64-v8a", 64, 8, 8, 8, device.LittleEndian},
 	{device.AndroidX86, "x86", 32, 4, 4, 4, device.LittleEndian},
-	{device.AndroidX86_64, "x86-64", 64, 8, 8, 8, device.LittleEndian},
+	{device.AndroidX86_64, "x86-64", 64, 8, 8, 4, device.LittleEndian},
```

Comment:
This test is exceptionally stupid - it is testing that constants defined in one place match those defined here. Happy to just delete this.